docker/worker/Dockerfile

```diff
@@ -24,9 +24,9 @@ RUN apt-get install -y dnsutils
 RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
 # Package Repo for Ubuntu 16.04
 #RUN curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list | tee /etc/apt/sources.list.d/msprod.list
-# Package Repo for Debian 8 (Docker Hub Python Image)
-RUN echo "deb [arch=amd64] https://packages.microsoft.com/debian/8/prod jessie main" >> /etc/apt/sources.list.d/msprod.list
-RUN apt-get update 
+# Package Repo for Debian 9 (Docker Hub Python Image)
+RUN echo "deb [arch=amd64] https://packages.microsoft.com/debian/9/prod "stretch" main" >> /etc/apt/sources.list.d/msprod.list
```

Comment:
I think eventually we should create our own python 3.5 base image, so that we don't have to worry about them updating to a different version of debian (which is why this check was failing, the worker image is now debian 9, not debian 8.